model/security/patch_role_mappings/structures.smithy

```diff
@@ -0,0 +1,21 @@
+// SPDX-License-Identifier: Apache-2.0
+//
+//  The OpenSearch Contributors require contributions made to
+//  this file be licensed under the Apache-2.0 license or a
+//  compatible open source license.
+
+$version: "2"
+namespace OpenSearch
+
+@input
+structure PatchRoleMappings_Input {
+    @required
+    @httpPayload
+    content: PatchRoleMappingStructure
```

Comment:
This operation directly takes a list:
```suggestion
    content: PatchOperationList
```
It is expected that this will cause build errors until #127 is merged